docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
   # The Unleash server contains the Unleash configuration and
   # communicates with server-side SDKs and the Unleash Proxy
   unleash:
-    image: ${UNLEASH_DOCKER_IMAGE:-unleashorg/unleash-server:latest}
+    image: test-enterprise
```

Comment:
I think you need to change this back :D

Reply:
You're very quick! I think I got that one already!